### PR TITLE
my first commit

### PR DESCRIPTION
**Pull request by vidhyasathyan**

### CODE REVIEW

File: HELP.md [hunk 1]
```diff
@@ -26,3 +26,5 @@ RUN  ./mvnw spring-boot:run from the project root folder
 if you get a maven wrapper not found error execute the below command and try running ./mvnw spring-boot:run
  mvn -N io.takari:maven:wrapper
 wrwer
+
+etetetetetet
```
Comment (arunplrgit) [inline HELP.md]: please make changes by modifying  this line

**Review by vidhyasathyan — COMMENTED** [review]
updated plz review

File: HELP.md (hunk 1 above)
Comment (arunplrgit) [reply HELP.md]: LGTM

**Review by arunplrgit — APPROVED** [review]
LGTM

**Pull request merged**